homeassistant/components/modbus/validators.py

```diff
@@ -203,6 +203,23 @@ def nan_validator(value: Any) -> int:
         raise vol.Invalid(f"invalid number {value}") from err
 
 
+def duplicate_fan_mode_validator(config: dict[str, Any]) -> dict:
```

Comment:
Can we type the dict?

Reply:
next version please, if you do not mind, because they are all going to change.

Reply:
version == PR